playbook/app/pb_kits/playbook/pb_title/_title.tsx

```diff
@@ -36,9 +39,25 @@ const Title = (props: TitleProps): React.ReactElement => {
   const ariaProps: {[key: string]: string | number} = buildAriaProps(aria)
   const dataProps: {[key: string]: string | number} = buildDataProps(data)
   const getBold = bold ? '' : 'thin'
+
+  const generateSizeClasses = () => {
+    let css = ''
+    
+    if (typeof size == "number" || typeof size == "string") {
```

Comment:
I think I would just make it a number, but I might be missing some kind of context

Reply:
Having it as just number would definitely make more sense @markdoeswork , but the kit right now allows both number or string as the size and we have to provide for both so we don't break current implementations